File: app/src/main/java/dev/ragnarok/fenrir/api/model/upload/UploadChatPhotoDto.java
package dev.ragnarok.fenrir.api.model.upload;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

public class UploadChatPhotoDto {

    @Nullable
    @SerializedName("response")
    public String response;

    @NonNull
    @Override
    public String toString() {
        return "UploadChatPhotoDto{" +
                "response='" + response + '\'' +
                '}';
    }
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/upload/UploadChatPhotoDto.kt
package dev.ragnarok.fenrir.api.model.upload

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class UploadChatPhotoDto {
    @SerialName("response")
    var response: String? = null
    override fun toString(): String {
        return "UploadChatPhotoDto{" +
                "response='" + response + '\'' +
                '}'
    }
}